### ISSUE TITLE
Convert to SWC template

### ISSUE THREAD

**Issue by fmichonneau** [datacarpentry/r-spatial-data-management-intro#42]
Would it be OK to convert this lesson to using the SWC lesson template instead of the custom one you are currently using? I'd be happy to take care of this conversion (unless someone else would like to do it).

**Comment by lwasser** [comment]
yes!! one minute @fmichonneau  this work has been started but wasn't completed - i think. does this repo look like the right template WITH a bunch of wrong stuff in there too 

https://github.com/lwasser/r-spatial-data-management-intro/tree/gh-pages/_episodes_rmd 

@butterflyology started to work on the conversion... and then i created a set of new rmd files with updated content

**Comment by lwasser** [comment]
we wanted to do this and had some momentum but then that died off... let's convert just let me know where i can help.  also see my PR...

**Comment by fmichonneau** [comment]
great! Yes I'll look into what you have started  and work from there.

**Comment by lwasser** [comment]
Awesome. i'm sorry it's a mess currently. we broke it up into parts and i worked on the content overhaul and Chris did the template part so i don't know a lot about the template but do know a lot about the old template that we are removing. :) Thanks. happy to see this moving forward.

**Comment by lwasser** [comment]
one last detail @tyson-swetnam  added one set of edits here

https://github.com/datacarpentry/r-spatial-data-management-intro/commit/1461f2df1146ac7d625b572027dd13f28dd7265d 

that would be good to keep. I"m happy to add them back in if that is necessary. There was a disconnect between how fast / slow i was moving to help with the template and folks starting to contribute! :)
